src/VisualStudio/Core/Def/Implementation/Snippets/AbstractSnippetExpansionClient.cs

```diff
@@ -169,7 +170,20 @@ private void CleanUpEndLocation(ITrackingSpan endTrackingSpan)
                 if (lineText.Trim() == string.Empty)
                 {
                     indentCaretOnCommit = true;
-                    indentDepth = lineText.Length;
+
+                    var document = this.SubjectBuffer.CurrentSnapshot.GetOpenDocumentInCurrentContextWithChanges();
+                    if (document != null)
+                    {
+                        var optionService = document.Project.Solution.Workspace.Services.GetService<IOptionService>();
+                        var tabSize = optionService.GetOption(FormattingOptions.TabSize, document.Project.Language);
+                        indentDepth = Microsoft.CodeAnalysis.Shared.Extensions.StringExtensions.GetColumnFromLineOffset(lineText, lineText.Length, tabSize);
```

Comment:
Add Using wasn't working here, because it already thought ```GetLineOffsetFromColumn``` was supposed to be the ITextSnapshotLineExtensions one. :disappointed: Adding the using manually works, of course.